### PR TITLE
Update 1.1.0 document for ibm cloud

### PR DESCRIPTION
**Pull request by adrian555**
This PR updates the install instructions to pick up the right version of kfdef file for Kubeflow deployment on IBM Cloud.

### CODE REVIEW

**Comment by adrian555** [comment]
cc @animeshsingh @Tomcli @shawnzhu

File: content/en/docs/ibm/create-cluster.md [hunk 1]
```diff
@@ -0,0 +1,138 @@
++++
+title = "Create an IBM Cloud cluster"
```
Comment (animeshsingh) [inline content/en/docs/ibm/create-cluster.md:2]: Would this have been better handled by point to offical docs from IKS? Else we will be updating often?

File: content/en/docs/ibm/deploy/install-kubeflow.md [hunk 2]
```diff
@@ -1,114 +1,30 @@
...
-weight = 4
+weight = 5
 +++
 
 This guide describes how to use the kfctl binary to deploy Kubeflow on IBM Cloud Kubernetes Service (IKS).
 
 ## Prerequisites
 
-### Installing the IBM Cloud developer tools
+* Logging in to IBM Cloud
```
Comment (animeshsingh) [inline content/en/docs/ibm/deploy/install-kubeflow.md:11]:
```suggestion
* Authenticate with IBM Cloud
```

File: content/en/docs/ibm/create-cluster.md (hunk 1 above)
Comment (adrian555) [reply content/en/docs/ibm/create-cluster.md:2]: Yes and no, the official docs definitely cover all the scenarios including creating clusters through UI. But usual times it is difficult to follow what exact command and options should be used. A up-to-date tutorial would be better than the current official docs. I meant something like https://cloud.ibm.com/docs/containers?topic=containers-cs_cluster_tutorial is better than https://cloud.ibm.com/docs/containers?topic=containers-getting-started.

The commands in this docs are easier to follow.

But yes, we can look for ways to improve.

File: content/en/docs/ibm/deploy/install-kubeflow.md (hunk 2 above)
Comment (adrian555) [reply content/en/docs/ibm/deploy/install-kubeflow.md:11]: Done

**Pull request merged**